### PR TITLE
Added 1st version of document

### PR DESCRIPTION
**Pull request by NT-D**
Add description about what you can learn and what tool we use in this project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,33 @@
-# ReactOnAzure
-This repository for developers who want to know how to host React app on Azure with security
+# React on Azure
+This repository for developers who want to know..
+- How to host React app on Azure
+- How to make connection secure between front-end and back-end with Azure features
+
+## Background
+SPA is popular in the web development world, but there is few documents/contents to describe how to host production app with Azure features. In addition to it, I and my customers are intersted it to host it on Azure, so we create this sample app.
```
Comment (sinnypan) [inline README.md:7]: [nit][grammar check] there **are** few documents/contents to ....

File: README.md [hunk 2]
```diff
@@ -1,2 +1,33 @@
...
+We uses following language and tools. As prerequistics, please read and try each tools tutorial.
+
+### Programming language
+- [TypeScript](https://www.typescriptlang.org/): JavaScript with type
+
+### Tools for single page application
+- [React](https://reactjs.org/): JavaScript library for building SPA.
+- [Jest](https://jestjs.io/): JavaScript testing framework. We use it for testing both React components and other codes.
+- [Enzyme](https://enzymejs.github.io/enzyme/): JavaScript testing utility for react. We use it for testing React components. This tool make us to write test for custom component easily.
+- [MSAL](https://github.com/AzureAD/microsoft-authentication-library-for-js): Microsoft Authentication library. We can get idToken with this SDK and Azure Active Directly. We want to utilize idToken for making secure coneection between backend.
```
Comment (sinnypan) [inline README.md:26]: [Typo]: We can get idToken with this SDK and Azure Active **Directory**. We want to utilize idToken for making secure **connection** between backend. 
[Question] We want to utilize idToken for making secure connection between backend **and what?**

File: README.md (hunk 1 above)
Comment (NT-D) [reply README.md]: Thanks for pointing out! Fixed

File: README.md (hunk 2 above)
Comment (NT-D) [reply README.md]: Fixed!  
A: Between back-end and front-end. I also  added `between front-end and back-end`.

I think GibHub still shows original version for your comment. Would you please see from [Commits] menu?

Comment (sinnypan) [reply README.md]: Yeah, I realized its different from AzDO, try to get used it lol. Thanks for the heads up

**Pull request merged**